src/Symfony/Component/HttpKernel/Tests/Bundle/BundleTest.php

```diff
@@ -13,9 +13,13 @@
 
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
 use Symfony\Component\HttpKernel\Tests\Fixtures\ExtensionNotValidBundle\ExtensionNotValidBundle;
 use Symfony\Component\HttpKernel\Tests\Fixtures\ExtensionPresentBundle\ExtensionPresentBundle;
 
+/**
+ * TODO Trigger class deprecation on version 5.1.
```

Comment:
Tests are not covered by backward compatibility promise.

Change the text with:
`Remove on version 6.0. While now it ensure BC layer.`